workspaces/github-actions/.changeset/light-swans-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-github-actions': patch
+---
+
+Updating readmes based on work done for github actions to work in portal
```

Comment:
This is a strange changeset, could you explain the changes that are being made instead?